### DISCUSSION TITLE
The issue of not being able to recognize PushType and DeliveryPriority from version 0.15.3 and onwards.

### DISCUSSION THREAD

**Discussion by Si-Hyeak-KANG** [relayrides/pushy#1057] [Q&A]
Hello,

I'm using the Live activity update library successfully. Currently, I'm using version 0.15.2.

However, when I try to upgrade to versions 0.15.3 and 0.15.4, there seems to be an issue with the push types.
The constructor parameters for SimpleApnsPushNotification no longer recognize DeliveryPriority and PushType.

What could be the reason behind this problem?

Thanks.

![image](https://github.com/jchambers/pushy/assets/79829085/48af6948-37b8-462f-814d-aeabbd0b743f)

<img width="400" alt="image" src="https://github.com/jchambers/pushy/assets/79829085/7532f63a-f9b1-4a88-8da1-26cae7f90455">

**Comment by jchambers** [comment]
In short: I don't know. I am unable to reproduce this problem locally. This sounds like a local development environment issue to me. It also sounds strikingly similar to #1055. One thing that stands out to me is that both of the things that you've highlighted in red are `enums`.

What IDE are you using? This is starting to sound like an IDE bug to me.

**Reply by josephyanks** [reply]
I also have this issue, and wasn't prior to upgrading past 15.2

It appears to be an issue with intellij, where it thinks it needs to use the FernFlower decompiler to see the classes. There's a few more classes affected, such as InterruptionLevel, LiveActivityEvent, and RejectionReason. 

If I find any fixes I will post back here.

**Reply by jchambers** [reply]
Thank you! That would be helpful!
